app/javascript/components/navbar/ExamNavbar.tsx

```diff
@@ -20,41 +22,58 @@ import Scratch from '@hourglass/containers/navbar/Scratch';
 import ExamMessages from '@hourglass/containers/navbar/ExamMessages';
 import AskQuestion from '@hourglass/components/navbar/AskQuestion';
 
-
 interface NavAccordionItemProps {
+  onSectionClick: (eventKey: string) => void;
+  expanded: boolean;
   Icon: IconType;
   label: string;
   className?: string;
   eventKey: string;
   direction?: 'up' | 'down';
 }
 
+const iconSize = '1.5em';
+
 export const NavAccordionItem: React.FC<NavAccordionItemProps> = (props) => {
   const {
+    onSectionClick,
+    expanded,
     Icon,
     label,
     eventKey,
     children,
     className = 'bg-secondary text-light',
     direction = 'down',
   } = props;
-  const iconSize = '1.5em';
+  // const dExpanded = displayClass(expanded);
```

Comment:
We can remove this now

Reply:
Yeah, I thought I'd done so already.  I must have only removed one of them, but this one was commented out and didn't get noticed.